profile/src/main/java/com/scottlogic/deg/profile/serialisation/ISerialiser.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 Scott Logic Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.scottlogic.deg.profile.serialisation;
+
+import java.io.IOException;
+
+public interface ISerialiser<TDto>
```

Comment:
In java/ datahelix we tend not to prefix interfaces with I (see https://stackoverflow.com/questions/541912/interface-naming-in-java). 

Additionally is this interface required?

Reply:
Yeh that was a mistake, I know you don't need it in Java but old habits. I removed the I.